src/scicookie/{{cookiecutter.project_slug}}/.pre-commit-config.yaml

```diff
@@ -7,145 +14,107 @@ repos:
-        (?x)(
-          docs
-        )
-      files: ""
+      entry: blue
       language: system
       pass_filenames: true
-      stages:
-        - commit
       types:
         - python
         - file
         - python
 {% endif -%}
 {%- if cookiecutter.use_black == "yes" %}
-    - entry: black
-      id: black
+    - id: black
       name: black
-      exclude: |
-        (?x)(
-          docs
-        )
-      files: ""
+      entry: black
       language: system
       pass_filenames: true
-      stages:
-        - commit
       types:
         - python
         - file
         - python
 {% endif %}
 {%- if cookiecutter.use_flake8 == "yes" %}
-    - entry: flake8
-      exclude: ^$
-      files: ""
-      id: flake8
-      language: python
+    - id: flake8
       name: flake8
+      entry: flake8
+      language: system
       pass_filenames: true
-      stages:
-        - commit
       types:
         - python
 {% endif %}
 {%- if cookiecutter.use_isort == "yes" %}
-    - entry: isort
-      exclude: "^.*/js/.*$"
-      files: ""
-      id: isort
-      language: python
+    - id: isort
       name: isort
+      entry: isort
+      language: system
       pass_filenames: true
-      stages:
-        - commit
       types:
         - python
 {% endif %}
 {%- if cookiecutter.use_mypy == "yes" %}
-    - entry: mypy
-      exclude: ^$
-      # note: it would be good to extend this to other apps as well
-      files: "{{ cookiecutter.package_slug }}/"
-      id: mypy
-      language: python
+    - id: mypy
       name: mypy
+      entry: mypy
+      language: system
+      files: "{{ package_path }}"
       pass_filenames: true
-      stages:
-        - commit
       types:
         - python
 {% endif %}
 {%- if cookiecutter.use_shellcheck == "yes" %}
-    - entry: shellcheck
-      id: shellcheck
-      language: system
+    - id: shellcheck
       name: shellcheck
-      files: .sh$
-      types:
-          - shell
+      entry: shellcheck
+      language: system
       types_or:
-          - file
-          - sh
-          - ash
-          - bash
-          - bats
-          - dash
-          - ksh
+        - sh
+        - shell
+        - ash
+        - bash
+        - bats
+        - dash
+        - ksh
 {% endif %}
 {%- if cookiecutter.use_bandit == "yes" %}
-    - entry: bandit
-      id: bandit
-      language: python
+    - id: bandit
       name: bandit
-      args: ['-iii', '-ll', .bandit]
+      entry: bandit
+      language: system
+      args: ['--configfile', 'pyproject.toml', '-iii', '-lll']
       pass_filenames: true
-      stages:
-        - commit
       types:
         - python
 {% endif %}
 {%- if cookiecutter.use_pydocstyle == "yes" %}
-    - entry: pydocstyle
-      exclude: ^$
-      files: "{{ cookiecutter.package_slug }}/"
-      id: pydocstyle
-      language: python
+    - id: pydocstyle
       name: pydocstyle
+      entry: pydocstyle
+      language: system
+      files: "{{ package_path }}"
```

Comment:
actually nop, because I create this variable in lowercase here:
https://github.com/osl-incubator/scicookie/pull/125/files#diff-e337bca5063014ffed5f8cf369c4b2f928ca860d881151e686bb6599bf62629eR2

we are using in uppercase in the other file .. because it is a python variable declared in the global scope of the module.

in this case it is a jinja2 variable